sandbox/settings/appleseed.studio.xml

```diff
@@ -10,6 +10,12 @@
         <parameter name="tile_ordering" value="spiral" />
     </parameters>
     <parameters name="ui">
+        <parameters name="file_dialog">
```

Comment:
Don't commit changes to this file.

Reply:
You still need to revert all your changes to this file.

Reply:
Sorry for that, every time after I run the Appleseed, this file is modified. I will check it carefully.